week2/index.js

```diff
@@ -0,0 +1,119 @@
+Added Abdullah
```

Comment:
Why do you have this line at the top?

Reply:
I added it when i was editing it from VIM editor in gitbash, but there is no need for it